core/template.js

```diff
@@ -990,6 +990,26 @@ var Template = Montage.specialize( /** @lends Template# */ {
                     argumentElement = templateArgumentProvider.getTemplateArgumentElement(
                         parameterName);
 
+                    if (parameterName === "*" && parameterElement.childElementCount > 0) { // has default
+                        if (argumentElement.childElementCount > 0 ||
+                            !!(argumentElement.firstChild &&
+                                argumentElement.firstChild.data &&
+                                argumentElement.firstChild.data.trim())
+                        ) { // has arguments
+                            var elementIds = this._getChildrenElementIds(parameterElement);
+                            
+                            for (var i = 0, length = elementIds.length; i < length; i++) {
+                                serialization.removeObject(elementIds[0]);
```

Comment:
Each call to this.removeObject() will cause a this.initWithObject(serializationObject), sounds inefficient. Maybe we should have instead a removeObjets() that takes an array and do the initWithObject(serializationObject) only once at the end of the loop?

Reply:
I provided a method called `removeObjectsWithLabels` for that purpose